
--
-- starts a Proxy with faulty script and a mock-backend that ACKs our login
--
chain_proxy('bug_41991-mock.py','bug_41991-test.py')



